queries.sql

```diff
@@ -1,2 +1,9 @@
-UPDATE teams
-    SET abbreviation 'WSH' = 'WAS'
\ No newline at end of file
+UPDATE teams SET abbreviation = 'WAS' WHERE abbreviation = 'WSH';
+
+SELECT startTime, homeScore, awayScore
```

Comment:
Great job getting the two checks so you don't end up with stuff from the day before.